Komodo was crashing at shutdown if the user performed a copy operation on a
Scintilla buffer before shutting down, bug 81293.

Index: gtk/ScintillaGTK.cxx
===================================================================
--- gtk/ScintillaGTK.cxx	(revision 27270)
+++ gtk/ScintillaGTK.cxx	(working copy)
@@ -78,9 +78,12 @@
 #pragma warning(disable: 4505)
 #endif
 
-#if GTK_CHECK_VERSION(2,6,0)
-#define USE_GTK_CLIPBOARD
-#endif
+// USE_GTK_CLIPBOARD is not used by Komodo as it is causing a crash upon
+// shutdown, see bug 81293. Clipboard operations still seem to work correctly
+// without this defined.
+//#if GTK_CHECK_VERSION(2,6,0)
+//#define USE_GTK_CLIPBOARD
+//#endif
 
 #if GLIB_MAJOR_VERSION < 2
 #define OBJECT_CLASS GtkObjectClass
